styles/style.css

```diff
@@ -1 +1,58 @@
-/* No CSS *//*# sourceMappingURL=style.css.map */
\ No newline at end of file
+#list {
```

Comment:
###  \[OPTIONAL\] Stretch Requirements Checklist & Common issues
- [ ] Do you mind improving the UI in terms of contrast? 😟 Currently, it is a bit of a struggle locating the input sections of the form.